unbalanced_dataset/base.py

```diff
@@ -181,3 +182,25 @@ def fit_sample(self, X, y):
         """
 
         return self.fit(X, y).sample(X, y)
+
+
+    def transform(self, X):
```

Comment:
Yep the problem of my testing was that my final estimator was not a `TransformerMixin` got it. I will correct that.